entrypoint.sh

```diff
@@ -19,9 +19,11 @@ case "$1" in
     ;;
     "pull-containers")
         # We can't to this in the Dockerfile, because the docker socket is not shared to there
-        exec /opt/FACT_core/src/install.py \
+        /opt/FACT_core/src/install.py \
             --backend-docker-images \
             --frontend-docker-images
+        [$? -eq 0] && printf "Sucessfully pulled docker containers. Please ignore the warning about unsupported distribution above.\n"
```

Comment:
Is this something we should fix in FACT_core? :sweat_smile:

Reply:
Most probably. I opened https://github.com/fkie-cad/FACT_core/issues/915 to track this.